cookbook/specs/stride.json

```diff
@@ -0,0 +1,1848 @@
+{
+    "proposal": {
+      "title": "Add Specs: Stride",
+      "description": "Adding new specification support for relaying Stride data on Lava",
+      "specs": [
+        {
+          "index": "STRIDE",
+          "name": "stride mainnet",
+          "enabled": true,
+          "imports": [
+            "COSMOSSDK"
```

Comment:
does is not support cosmwasm?
COSMOSSDKFULL -> with cosmwasm
COSMOSSDK -> without cosmwasm

Reply:
Nope Stride doesn't support wasm